Soft_Skills/README.md

```diff
@@ -8,3 +8,28 @@ Improve your soft skills and get ready for interviews
 
 * [Linkedin](linkedin)   
     Improve your soft skills and get ready for interviews
+    
+# Body Language
+
+* [Body Langauge Ted Talk](https://www.ted.com/talks/amy_cuddy_your_body_language_may_shape_who_you_are/transcript)
+A powerfull explantion on how your body language shape who you are.
+
+* [Body Langauge The Power Of Hand Ted Talk](https://www.youtube.com/watch?v=ZZZ7k8cMA-4)
+A talk that talks about the power of your hands body language.
+
+* [How To Speak Ted Talk](https://www.youtube.com/watch?v=eIho2S0ZahI&t=465s)
+A storng video about how to talk that people would want to listen to you.
+
+* how to talk with your hands:  
+[Hand Gestures Articale](https://www.scienceofpeople.com/hand-gestures/)      
+[Hand Shake Tips](https://www.forbes.com/sites/carolkinseygoman/2011/09/28/shaking-hands-body-language-at-work/#405558e04c4e)  
+[More About Hand Gestures](https://westsidetoastmasters.com/resources/book_of_body_language/chap2.html)
+
+# Other
```

Comment:
Since both the added resources have 'developer'in their title, maybe the section has a better name than 'other'?